src/lib/update-notifications/index.tsx

```diff
@@ -0,0 +1,72 @@
+// istanbul ignore file: Covered by E2E
+
+import { qs, qsMaybe } from '@lib/util/dom';
+import { parseVersion, versionLessThan } from '@lib/util/versions';
+
+import CHANGELOG_URL from 'consts:changelog-url';
+import USERSCRIPT_FEATURE_HISTORY from 'consts:userscript-feature-history';
+import USERSCRIPT_ID from 'consts:userscript-id';
+import bannerStyle from './banner.scss';
+
+const LAST_DISPLAYED_KEY = `ROpdebee_${USERSCRIPT_ID}_last_notified_update`;
+
+export function maybeDisplayNewFeatures(): void {
+    const lastDisplayedVersionString = localStorage.getItem(LAST_DISPLAYED_KEY);
+    // eslint-disable-next-line no-restricted-globals
+    const scriptInfo = GM.info.script;
+
+    // If we've never shown a notification before, don't show one. Otherwise
+    // a couple of years down the line, someone might get spammed with a huge
+    // banner if we've made many updates.
+    if (!lastDisplayedVersionString) {
+        // Set the userscript version so that the next update will be displayed.
+        localStorage.setItem(LAST_DISPLAYED_KEY, scriptInfo.version);
+        return;
+    }
+
+    const lastDisplayedVersion = parseVersion(lastDisplayedVersionString);
+    const newFeatures = USERSCRIPT_FEATURE_HISTORY
+        .filter((feat) => versionLessThan(lastDisplayedVersion, parseVersion(feat.versionAdded)));
+
+    showFeatureNotification(scriptInfo.name, scriptInfo.version, newFeatures.map((feat) => feat.description));
```

Comment:
> I had a quick read only and did not test

Derp, the notification gets also shown when there are no new features and when it has already been dismissed. We are missing a second early exit condition here...
You only have two load a page where the userscript is active twice and the following will appear:
![image](https://user-images.githubusercontent.com/52860029/172832525-f66e9fa5-a8ba-431b-bffe-89b46bdbdd52.png)

Reply:
#462 

You know what the worst part is? I had already noticed it while testing yesterday, but for some reason it didn't occur to me at all that this was a bug. 🤦